### ISSUE TITLE
layout issue on modal

### ISSUE THREAD

**Issue by shawntaylor** [alexgorbatchev/jquery-textext#150]
I am encountering a layout issue when I load this awesome plugin in a bootstrap modal... or more specifically, when I load it a 2nd+ time, and only when running on my local network windows server (it's an internal app - using web-server as dev for ease of sharing).

Here is a screencast showing the error: http://screencast.com/t/zhoITnHcK
As you can see here, the height and padding are not being set by the plugin as they should be: http://screencast.com/t/g4oHt2yCpZ.

Error exists in all browsers. After a few hours (at least, haven't measured the time yet) the issue resets and I can load the proper layout a single time, and then it's back to the messed up layout for every subsequent load.

Any ideas? Please help!

**Comment by DmitryIvanovIAMM** [comment]
Yes, I also see wrong initialization problem of Textext in modal window. What I do: just add call of invalidateBounds for textext after modal was shown:
    $("#myModal").on('shown.bs.modal', function(){
        $('#textarea').textext()[0].invalidateBounds();
    });

After this textext correctly calculates sizes of visual elements.
Thanks
